silhouette/test/com/mohiva/play/silhouette/impl/providers/state/UserStateItemHandlerSpec.scala

```diff
@@ -0,0 +1,73 @@
+package com.mohiva.play.silhouette.impl.providers.state
```

Comment:
The file and class level doc comments are missing. It has also the same formatting issues as the above file.